pints/toy/_toy_classes.py

```diff
@@ -61,3 +64,134 @@ def suggested_times(self):
         """
         raise NotImplementedError
 
+    def simulate(self, parameters, times):
+        """ See :meth:`pints.ForwardModel.simulate()`. """
+        raise NotImplementedError
+
+
+class ToyODEModel(ToyModel):
```

Comment:
This implements some of the `ForwardModelS1` methods, and so should extend that class?

Unless we want to split into toy ode models that do and don't do derivatives, but I'm happy to only have one that does ODEs where derivatives are available, as long as we make that clear in the docstring!

Reply:
I've now updated things as we discussed in the meeting, with a separate ToyODEModel class.